src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,500;1,700&display=swap" rel="stylesheet">
+
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <div class="card__content" >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__page"
+        >
+      </div>
+
+      <div class="card__main-info">
+        <p class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__product-code" >Product code: 195434</p>
+
+        <div class="card__container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__costs">
+          <p class="card__cost card__cost-title--style_smoll">Price:</p>
+          <p class="card__cost card__cost-content--style_bold">$2,199</p>
+        </div>
+      </div>
+
+      <div class="card__button-section card__button-section--hover">
+        <a
+          href="#"
+          class="card__link card__link--hover"
+          data-qa="hover"
+        ><button type="submit" class="card__button">Buy</button></a>
```

Comment:
don't use `button` inside `a` tag. just leave `a` and add proper styles
```suggestion
        <a
          href="#"
          class="card__link card__link--hover"
          data-qa="hover"
        >
          Buy
        </a>
```